app/index.js

```diff
@@ -290,6 +293,14 @@ app.on('ready', () => installDevExtensions(isDev).then(() => {
       Menu.getApplicationMenu().popup(Math.ceil(x), Math.ceil(y));
     });
 
+    rpc.on('open context menu', () => {
+      const commands = getKeymaps().commands;
+      const contextMenuTemplate = editMenu(commands).submenu
+        .concat({type: 'separator'}, shellMenu(commands, createWindow).submenu);
+      const contextMenu = Menu.buildFromTemplate(contextMenuTemplate);
+      setTimeout(() => contextMenu.popup(), 100);
```

Comment:
Why do you need this `setTimeout`? If you really need this, you should add a comment to explain why.